draft-ietf-quic-transport.md

```diff
@@ -404,26 +401,44 @@ Long Packet Type:
   indicate one of 128 packet types.  The types specified for this version are
   listed in {{long-packet-types}}.
 
-Connection ID:
+DCIL and SCIL:
+
+: Octet 1 contains the lengths of the two connection ID fields that follow it.
+  These lengths are encoded as two 4-bit unsigned integers. The Destination
+  Connection ID Length (DCIL) field occupies the 4 high bits of the octet and
+  the Source Connection ID Length (SCIL) field occupies the 4 low bits of the
+  octet.  An encoded length of 0 indicates that the connection ID is also 0
+  octets in length.  Non-zero encoded lengths are increased by 3 to get the full
+  length of the connection ID, producing a length between 4 and 18 octets
+  inclusive.  For example, an octet with the value 0x50 describes an 8 octet
+  Destination Connection ID and a zero octet Source Connection ID.
 
-: Octets 1 through 8 contain the connection ID. {{connection-id}} describes the
-  use of this field in more detail.
+Destination Connection ID:
+
+: The Destination Connection ID field starts at octet 2 and is either 0 octets
+  in length or between 4 and 18 octets. {{connection-id}} describes the use of
+  this field in more detail.
+
+Source Connection ID:
+
+: The Source Connection ID field starts at octet 2 and is either 0 octets in
```

Comment:
They can't both start at octet 2, unless the destination Connection ID is 0 length.

Reply:
+1. I would remove the starting position.